DungeonTools.lua

```diff
@@ -800,6 +800,21 @@ function MDT:CreateMenu()
     highlight:SetPoint("TOPRIGHT", resizer, -6, 0)
     resizer:SetHighlightTexture(highlight)
 
+    -- Lock Enemy data on map
+    self.main_frame.lockButton = CreateFrame("Button", "DungeonToolsLockEnemyButton", self.main_frame, "UIPanelCloseButton")
+    db.mapEnemiesLocked = db.mapEnemiesLocked or false
```

Comment:
Same as above

Reply:
That makes complete sense. I'll make the requested changes.